From a581c65b2d4d17d576f4300f956d2e5c9dc2475f Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Sat, 3 Nov 2018 11:27:42 +0100
Subject: [PATCH] pie-gsi tracking

commit 16e322265060626979f9a43c1e3166d31a146021
Merge: f39c3f1 487ba45
Author: Treehugger Robot <treehugger-gerrit@google.com>
Date:   Sat Nov 3 07:33:28 2018 +0000

    Merge "Ensure all android.process.media apps have usesCleartextTraffic=true" into pie-gsi

commit 487ba4587ace3d328678aa69809ad48e9e095b61
Author: Chad Brubaker <cbrubaker@google.com>
Date:   Mon Sep 17 10:23:57 2018 -0700

    Ensure all android.process.media apps have usesCleartextTraffic=true

    Change-Id: Ibed8dbf680895092a857220a613a42d29320d5ae
    Merged-In: Ibed8dbf680895092a857220a613a42d29320d5ae
    Fixes: 113533479
    Test: DownloadManager CTS
    (cherry picked from commit 5e0e179554e2357bfc55f10454a64f1aea50c2ba)
    (cherry picked from commit 0e474d9ea30e3a2c54531ff5560c5103ba0b1248)

Change-Id: I03f5ad03598fe8cda0226bd46f69e038382acff0
---
 AndroidManifest.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index a1ae9bb..5afdcc5 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -20,7 +20,8 @@
                  android:label="@string/app_label"
                  android:icon="@mipmap/ic_launcher"
                  android:allowBackup="false"
-                 android:supportsRtl="true">
+                 android:supportsRtl="true"
+                 android:usesCleartextTraffic="true">
         <provider android:name="MediaProvider" android:authorities="media"
                 android:multiprocess="false" android:exported="true">
             <grant-uri-permission android:pathPrefix="/external/" />
-- 
2.17.1

